internal/tenantsvc/service.go

```diff
@@ -111,6 +112,51 @@ func (t *TenantService) CreateTenant(ctx context.Context, req *pb.CreateTenantRe
 	return t.createOrUpdateTenant(ctx, req.Tenant, false)
 }
 
+// UpdateTenant handles tenant updation requests.
+func (t *TenantService) UpdateTenant(ctx context.Context, req *pb.UpdateTenantRequest) (*pb.Tenant, error) {
+	m, err := t.rdb.HGetAll(tenantKey(req.TenantName)).Result()
+	if err != nil {
+		return nil, err
+	}
+
+	if len(m) == 0 {
+		return nil, ErrTenantNotFound
+	}
+
+	newFlag := strconv.FormatBool(req.Approvesdc)
+	if err != nil {
+		return nil, err
+	}
+
+	existingFlag, err := t.rdb.HGet(tenantKey(req.TenantName), "Approvesdc").Result()
+	if err != nil {
+		return nil, err
+	}
+
+	if strings.EqualFold(newFlag, existingFlag) {
+		return nil, err
+	}
+
+	_, err = t.rdb.HSet(tenantKey(req.TenantName), "Approvesdc", newFlag).Result()
```

Comment:
Can this be changed to `approve_sdc` to align with the formatting of existing fields?

Reply:
changed to approve_sdc